drv/minibar-ignition-server/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "drv-minibar-ignition-server"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+cfg-if = { workspace = true }
+idol-runtime = { workspace = true }
+num-traits = { workspace = true }
+zerocopy = { workspace = true }
+zerocopy-derive = { workspace = true }
+
+drv-fpga-api = { path = "../fpga-api" }
+drv-ignition-api = { path = "../ignition-api" }
+drv-minibar-seq-server = { path = "../minibar-seq-server" }
```

Comment:
Are all of these dependencies necessary?  I'm suspicious about `drv-minibar-seq-server`

Reply:
`drv-minibar-seq-server` was debris from a discarded path.